packages/components/package.json

```diff
@@ -26,7 +26,8 @@
     "build-storybook": "build-storybook"
   },
   "files": [
-    "dist"
+    "dist",
+    "src/typings.d.ts"
   ],
```

Comment:
If you're having trouble with this .d.ts file you might want to just rename it to `typings.ts`. `.d.ts` files are kind of weird to have in source code. For example the typescript compiler doesn't even copy them over to the output directory. If you rename it to `.ts` the compiler will produce a `.d.ts` for you anyways

Reply:
alright!

Reply:
done